mypy/test/testpep561.py

```diff
@@ -67,10 +67,18 @@ def virtualenv(self,
                 yield venv_dir, os.path.abspath(os.path.join(venv_dir, 'bin', 'python'))
 
     def install_package(self, pkg: str,
-                        python_executable: str = sys.executable) -> None:
+                        python_executable: str = sys.executable,
+                        use_pip: bool = True,
+                        editable: bool = False) -> None:
         """Context manager to temporarily install a package from test-data/packages/pkg/"""
         working_dir = os.path.join(package_path, pkg)
-        install_cmd = [python_executable, '-m', 'pip', 'install', '.']
+        if use_pip:
+            install_cmd = [python_executable, '-m', 'pip', 'install']
+            if editable:
+                install_cmd.append('-e')
+            install_cmd.append('.')
+        else:
+            install_cmd = [python_executable, 'setup.py', 'install']
```

Comment:
Should this be `develop` if `editable` is True?

Reply:
I suppose that would be better, but the test should just use pip and editable (since `pip install -e .` calls `python setup.py develop`). But just to be safe, I will change it.